crypto/keyring/legacy_test.go

```diff
@@ -27,7 +28,7 @@ func TestLegacyKeybase(t *testing.T) {
 
 	kb, err := keyring.NewLegacy("keys", filepath.Join(dir, "keys"))
 	require.NoError(t, err)
-	t.Cleanup(func() { kb.Close() })
+	t.Cleanup(func() { assert.NoError(t, kb.Close()) })
```

Comment:
```suggestion
	t.Cleanup(func() { assert.NoError(t, kb.Close()) })
```

Why? Again, the error returned by `Close()` is likely unrelated to the test.

Please add a `// nolint: errcheck` - that would make more sense.